modules/gitops-argo-cd-properties.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="argo-cd-properties_{context}"]
+= Argo CD custom resource properties
+
+[role="_abstract"]
+The ArgoCD Custom Resource consists of the following properties:
+
+|===
+|**Name** |**Default** |**Description**
```

Comment:
If the values such as `[Object]` or `[empty]` are to be provided by the reader, then they can be documented according to the guidance in https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values : for example, `__<Object>__`, `__<empty>__`, and `__<default_Argo_CD_Known_Hosts>__`